src/PosteriorStats.jl

```diff
@@ -2,20 +2,31 @@ module PosteriorStats
 
 using Compat: @constprop
 using DataInterpolations: DataInterpolations
-using Distributions: Distributions
+using Distributions: cov, Distributions, invcov, mean
```

Comment:
For clarity, I prefer to (in most cases) prefix functions from other packages with the package name:
```suggestion
using Distributions: Distributions
```